### PR TITLE
chore(rxOptionTable): decompose monolithic source

### PR DESCRIPTION
**Pull request by CITguy**
JIRA: https://jira.rax.io/browse/FRMW-358

### LGTMS
- [x] Dev LGTM
- [x] Dev LGTM

### CODE REVIEW

File: src/rxOptionTable/rxOptionTable.spec.js [hunk 1]
```diff
@@ -1,326 +0,0 @@
...
-
-        // Inject in angular constructs
-        inject(function ($location, $rootScope, $compile) {
-            rootScope = $rootScope;
-            scope = $rootScope.$new();
-            compile = $compile;
-        });
-    });
-
-    describe('rxOptionTable directive (checkbox)', function () {
```
Comment (CITguy) [inline src/rxOptionTable/rxOptionTable.spec.js]: moved to `src/rxOptionTable/scripts/rxOptionTable.spec.js`

File: src/rxOptionTable/rxOptionTable.spec.js [hunk 2]
```diff
@@ -1,326 +0,0 @@
...
-
-            checkTableScope.selectAll();
-
-            expect(checkTableScope.model[0]).to.be.true;
-            expect(checkTableScope.model[1]).to.be.true;
-
-        });
-    });
-
-    describe('rxOptionTable directive (radio)', function () {
```
Comment (CITguy) [inline src/rxOptionTable/rxOptionTable.spec.js]: moved to `src/rxOptionTable/scripts/rxOptionTable.spec.js`

File: src/rxOptionTable/rxOptionTable.spec.js [hunk 3]
```diff
@@ -1,326 +0,0 @@
...
-        });
-
-        it('should correctly interpolate expression and apply the currency filter to it', function () {
-            var column = { key: '{{ amount | currency }}' },
-                data = { amount: 12.5 };
-            expect(elScope.getContent(column, data)).to.be.eq('$12.50');
-        });
-    });
-
-    describe('deprecated rxFormOptionTable directive', function () {
```
Comment (CITguy) [inline src/rxOptionTable/rxOptionTable.spec.js]: moved to `src/rxOptionTable/scripts/rxFormOptionTable.spec.js`

**Comment by 100stacks** [comment]
LGTM

**Comment by bry4n** [comment]
LGTM

**Pull request merged**